.github/workflows/push-to-prod-draft.yml

```diff
@@ -25,12 +25,18 @@ jobs:
           sed -i -E "s/\/dev\/i2c-1:\/dev\/i2c-1/\/dev\/i2c-7:\/dev\/i2c-7/g" docker-compose.yml
           ROCKPI="-rockpi"
           echo "ROCKPI=$ROCKPI" >> $GITHUB_ENV
-      - name: Balena Deploy
+      - name: Balena Deploy (ROCK Pi)
+        if: matrix.sbc == 'rockpi' && success()
         uses: nebraltd/balena-cli-action@v12.54.5
-        if: success()
         with:
           balena_api_token: ${{secrets.BALENA_API_TOKEN}}
```

Comment:
Just making sure you meant to leave this unnumbered `BALENA_API_TOKEN`.

Reply:
i was starting from 0 ... so there is 4 tokens BALENA_API_TOKEN and then the same but with _1, _2 and _3 appended.